translator/tocwconfig/tocwconfig_test.go

```diff
@@ -327,19 +327,64 @@ func TestPrometheusConfig(t *testing.T) {
 	expectedEnvVars := map[string]string{}
 	tokenReplacements := map[string]string{
 		prometheusFileNameToken: strings.ReplaceAll(prometheusConfigFileName, "\\", "\\\\"),
-		ecsSdFileNamToken:       strings.ReplaceAll(ecsSdFileName, "\\", "\\\\"),
+		ecsSdFileNameToken:      strings.ReplaceAll(ecsSdFileName, "\\", "\\\\"),
 	}
 	// Load prometheus config and replace ecs sd results file name token with temp file name
-	prometheusConfig = strings.ReplaceAll(prometheusConfig, "{"+ecsSdFileNamToken+"}", ecsSdFileName)
+	testPrometheusConfig := strings.ReplaceAll(prometheusConfig, "{"+ecsSdFileNameToken+"}", ecsSdFileName)
 	// Write the modified prometheus config to temp prometheus config file
-	err := os.WriteFile(prometheusConfigFileName, []byte(prometheusConfig), os.ModePerm)
+	err := os.WriteFile(prometheusConfigFileName, []byte(testPrometheusConfig), os.ModePerm)
 	require.NoError(t, err)
 	// In the following checks, we first load the json and replace tokens with the temp files
 	// Additionally, before comparing with actual, we again replace tokens with temp files in the expected toml & yaml
 	checkTranslation(t, "prometheus_config_linux", "linux", expectedEnvVars, "", tokenReplacements)
 	checkTranslation(t, "prometheus_config_windows", "windows", nil, "", tokenReplacements)
 }
 
+func TestOtelPrometheusConfig(t *testing.T) {
```

Comment:
This test is currently failing. Think you just need to add https://github.com/aws/amazon-cloudwatch-agent/blob/a546de303015a0f6617ca6976690e9cf471892db/translator/tocwconfig/tocwconfig_unix_test.go#L37